mllib/src/main/scala/org/apache/spark/ml/feature/LSH.scala

```diff
@@ -106,22 +106,24 @@ private[ml] abstract class LSHModel[T <: LSHModel[T]]
    * transformed data when necessary.
    *
    * This method implements two ways of fetching k nearest neighbors:
-   *  - Single Probing: Fast, return at most k elements (Probing only one buckets)
-   *  - Multiple Probing: Slow, return exact k elements (Probing multiple buckets close to the key)
+   *  - Single-probe: Fast, return at most k elements (Probing only one buckets)
+   *  - Multi-probe: Slow, return exact k elements (Probing multiple buckets close to the key)
+   *
+   * Currently it is made private since more discussion is needed for Multi-probe
```

Comment:
Sorry, I meant to make the `approxNearestNeighbors` public in Line 163 LSH.scala. I copied all the docs there except the difference of SingleProbe/MultiProbe.